test/uk/gov/hmrc/economiccrimelevyregistration/connectors/TaxEnrolmentsConnectorSpec.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.economiccrimelevyregistration.connectors
+
+import org.mockito.ArgumentMatchers
+import org.mockito.ArgumentMatchers.any
+import uk.gov.hmrc.economiccrimelevyregistration.base.SpecBase
+import uk.gov.hmrc.economiccrimelevyregistration.models.eacd.CreateEnrolmentRequest
+import uk.gov.hmrc.http.{HttpClient, HttpResponse}
+import com.danielasfregola.randomdatagenerator.RandomDataGenerator.derivedArbitrary
+import uk.gov.hmrc.economiccrimelevyregistration.models.eacd.CreateEnrolmentRequest.serviceName
+
+import scala.concurrent.Future
+
+class TaxEnrolmentsConnectorSpec extends SpecBase {
+  val mockHttpClient: HttpClient = mock[HttpClient]
+  val connector                  = new TaxEnrolmentsConnector(appConfig, mockHttpClient)
+  val taxEnrolmentsUrl           = s"${appConfig.taxEnrolmentsBaseUrl}/tax-enrolments/service/$serviceName/enrolment"
```

Comment:
can probably just inline this in the test as there is only one test?

Reply:
Done